urls.py

```diff
@@ -40,7 +40,8 @@
                 path("pyconcz-discord/", RedirectView.as_view(url='https://discord.gg/UtPkpPB7te', permanent=True)),
                 path("discord/", RedirectView.as_view(url='https://discord.gg/UtPkpPB7te', permanent=True)),
 
-                path("program/schedule/", TemplateView.as_view(template_name='pages/schedule.html'), name='schedule'),
+                # Note: should we keep the temporary schedule?
+                # path("program/schedule/", TemplateView.as_view(template_name='pages/schedule.html'), name='schedule'),
```

Comment:
@benabraham Zatím jsem zakomentoval view s dočasným schedule, aby fungoval redirect. Pokud budeme na produkci chtít zatím zobrazovat ten dočasný, tak by mělo stačit odkomentovat (v configu je dříve, než ten redirect).

Reply:
Jasně, díky. Dočasný pojede jen teď, než to naplníme a pak to přehodíme a smažeme úplně.